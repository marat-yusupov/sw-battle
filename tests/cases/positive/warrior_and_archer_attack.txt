CREATE_MAP 5 5 

SPAWN_WARRIOR 1 1 3 5 2

SPAWN_ARCHER 2 5 3 5 2 1 4