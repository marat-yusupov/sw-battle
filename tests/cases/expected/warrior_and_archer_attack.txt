[1] MAP_CREATED width=5 height=5 
[1] UNIT_SPAWNED unitId=1 unitType=Warrior x=1 y=3 
[1] UNIT_SPAWNED unitId=2 unitType=Archer x=5 y=3 
[2] UNIT_ATTACKED attackerUnitId=2 targetUnitId=1 damage=2 targetHp=3 
[3] UNIT_ATTACKED attackerUnitId=2 targetUnitId=1 damage=2 targetHp=1 
[4] UNIT_ATTACKED attackerUnitId=2 targetUnitId=1 damage=2 targetHp=0 
[4] UNIT_DIED unitId=1 
